addons/sale/views/sale_portal_templates.xml

```diff
@@ -586,7 +586,7 @@
             <section t-if="sale_order.payment_term_id" class="mt-5">
                 <h3 class="">Payment terms</h3>
                 <hr class="mt-0 mb-1"/>
-                <span t-field="sale_order.payment_term_id"/>
+                <span t-field="sale_order.payment_term_id.note"/>
```

Comment:
Unrelated diff.
Please clean your diff before removing the request for review :p

Reply:
This wasn't meant to be reviewed already. C3POdoo was too fast. I'll put the PR in draft back sorry :(

Reply:
It looks like you inadvertently squashed your commit with https://github.com/odoo/odoo/commit/990aae55b1a2b95765a848b6fc122bb69f768ce1